examples/src/fibonacci/fib2/air.rs

```diff
@@ -69,3 +68,57 @@ impl Air for FibAir {
         ]
     }
 }
+
+#[derive(Clone, Default)]
+pub struct PlainLogUpGkrEval<B: FieldElement> {
```

Comment:
We should export a `DummyLogUpGkrEval<BaseField, PublicInputs>` can be used when LogUp-GKR is disabled (instead of reimplementing it everytime)

Reply:
Good idea!
Implemented and exported